.coveragerc

```diff
@@ -414,6 +414,8 @@ omit =
     homeassistant/components/habitica/__init__.py
     homeassistant/components/habitica/const.py
     homeassistant/components/habitica/sensor.py
+    homeassistant/components/halohome/__init__.py
+    homeassistant/components/halohome/light.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.